winui/Masked-TextBox/Customization.md

```diff
@@ -7,4 +7,81 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Customization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Customization in WinUI MaskedTextBox
```

Comment:
@Vignesh-SF3580 Learn all about the customization such as header, header template, and description support in the MaskedTextBox (SfMaskedTextBox) control.

Reply:
Changed the description.